src/Microsoft.AspNet.StaticFiles/IDirectoryFormatter.cs

```diff
@@ -14,6 +15,11 @@ namespace Microsoft.AspNet.StaticFiles
     public interface IDirectoryFormatter
     {
         /// <summary>
+        /// HtmlEncoder to be used to encode content.
+        /// </summary>
+        IHtmlEncoder HtmlEncoder { get; set; }
```

Comment:
I don't recommend putting this on the interface, HTML is only one possible output.